index.bs

```diff
@@ -783,7 +783,22 @@ spec:css-syntax-3;
     6.  Let |sameOriginWithAncestors| be `true` if |settings| is [=same-origin with its
         ancestors=], and `false` otherwise.
 
-    7.  Run the following steps [=in parallel=]:
+    7.  Let |key| be the result of [=map/getting the key=] of |options|.
+
+    8.  If |key| is {{CredentialRequestOptions/publicKey}} then
```

Comment:
We should add web OTP

https://wicg.github.io/web-otp/#sctn-permissions-policy

This mechanism is a bit of a pain to extend to other credential types and somewhat defeats the purpose of the credential types registry. Can we add the permission policy names to the registry, and then iterate those?

Reply:
Good idea, agreed. Although I suggest we:
1. submit an issue about this.   ( done: see issue #184 )
2. address it in a separate PR **_after both_** this PR and [PR 181](https://github.com/w3c/webappsec-credential-management/pull/181) (which creates the registry) land.